server.go

```diff
@@ -1767,7 +1768,11 @@ func (s *Server) executeShowSeriesStatement(stmt *influxql.ShowSeriesStatement,
 	if stmt.Source != nil {
 		// TODO: handle multiple measurement sources
 		if m, ok := stmt.Source.(*influxql.Measurement); ok {
-			measurements = append(measurements, m.Name)
+			segments, err := influxql.SplitIdent(m.Name)
+			if err != nil {
+				return &Result{Err: err}
+			}
+			measurements = append(measurements, segments[2])
```

Comment:
Ben added a change to normalize measurements (e.g. "cpu" becomes "db"."policy"."cpu") in the AST.  `influxql.SplitIdent` splits a normalized measurement into it's pieces.  The measurements map uses only the measurement name as the key, so it has to be split out here.